src/ui/public/vis/map/_leaflet_overrides.scss

```diff
@@ -0,0 +1,126 @@
+// SASSTODO: Create these tooltip variables in EUI
+// And/Or create a tooltip mixin
+$temp-euiTooltipBackground: tintOrShade($euiColorFullShade, 25%, 90%);
+$temp-euiTooltipText: $euiColorGhost;
+
+// Converted leaflet icon sprite into background svg for custom coloring (dark mode)
+$visMapLeafletSprite: "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 540 60' height='60' width='540'%3E%3Cg fill='#{hexToRGB($euiTextColor)}'%3E%3Cg%3E%3Cpath d='M18 36v6h6v-6h-6zm4 4h-2v-2h2v2z'/%3E%3Cpath d='M36 18v6h6v-6h-6zm4 4h-2v-2h2v2z'/%3E%3Cpath d='M23.142 39.145l-2.285-2.29 16-15.998 2.285 2.285z'/%3E%3C/g%3E%3Cpath d='M100 24.565l-2.096 14.83L83.07 42 76 28.773 86.463 18z'/%3E%3Cpath d='M140 20h20v20h-20z'/%3E%3Cpath d='M221 30c0 6.078-4.926 11-11 11s-11-4.922-11-11c0-6.074 4.926-11 11-11s11 4.926 11 11z'/%3E%3Cpath d='M270,19c-4.971,0-9,4.029-9,9c0,4.971,5.001,12,9,14c4.001-2,9-9.029,9-14C279,23.029,274.971,19,270,19z M270,31.5c-2.484,0-4.5-2.014-4.5-4.5c0-2.484,2.016-4.5,4.5-4.5c2.485,0,4.5,2.016,4.5,4.5C274.5,29.486,272.485,31.5,270,31.5z'/%3E%3Cg%3E%3Cpath d='M337,30.156v0.407v5.604c0,1.658-1.344,3-3,3h-10c-1.655,0-3-1.342-3-3v-10c0-1.657,1.345-3,3-3h6.345 l3.19-3.17H324c-3.313,0-6,2.687-6,6v10c0,3.313,2.687,6,6,6h10c3.314,0,6-2.687,6-6v-8.809L337,30.156'/%3E%3Cpath d='M338.72 24.637l-8.892 8.892H327V30.7l8.89-8.89z'/%3E%3Cpath d='M338.697 17.826h4v4h-4z' transform='rotate(-134.99 340.703 19.817)'/%3E%3C/g%3E%3Cg%3E%3Cpath d='M381 42h18V24h-18v18zm14-16h2v14h-2V26zm-4 0h2v14h-2V26zm-4 0h2v14h-2V26zm-4 0h2v14h-2V26z'/%3E%3Cpath d='M395 20v-4h-10v4h-6v2h22v-2h-6zm-2 0h-6v-2h6v2z'/%3E%3C/g%3E%3C/g%3E%3C/svg%3E%0A";
+
+.leaflet-bar {
+  @include euiBottomShadowMedium($color: $euiShadowColorLarge, $opacity: .1)
```

Comment:
I find that new shadow rather hard to distinguish from the background on maps:

**Before**:
![screenshot-20181017-111144](https://user-images.githubusercontent.com/877229/47075766-7f736480-d1fd-11e8-9fe3-e86cb14ec583.png)

**After**:
![screenshot-20181017-111148](https://user-images.githubusercontent.com/877229/47075777-85694580-d1fd-11e8-8cd6-bb935b1d0fd0.png)

Maybe we should give it slightly more opacity? But just a suggestion, and in the end maybe @thomasneirynck could look at the map changes, since it seem to be the most visual changes due to this PR.

Reply:
I can increase the shadow a bit, but the original was way to dark.